client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -77,7 +77,7 @@ function ReaderSubscriptionListItem( {
 				<div>{ siteExcerpt }</div>
 				<div className="reader-subscription-list-item__site-url">
 					<a href={ siteUrl }> { siteUrl && stripUrl( siteUrl ) } </a>
-					{ moment( lastUpdated ).fromNow() }
```

Comment:
included this line so that we don't display a lastUpdated data if it doesn't exist